neo/Settings.cs

```diff
@@ -14,6 +14,8 @@ internal class Settings
         public string[] SeedList { get; private set; }
         public IReadOnlyDictionary<TransactionType, Fixed8> SystemFee { get; private set; }
         public Fixed8 LowPriorityThreshold { get; private set; }
+        public int MaxTransactionsPerBlock { get; private set; }
+        public int MaxFreeTransactionsPerBlock { get; private set; }
```

Comment:
These two parameters and policy settings may be different, it is best to have a way to reuse.

Reply:
Since the core needs to know these, maybe the plugin policy should read these from the the core.

Reply:
I suddenly thought that maybe only the consensus nodes need to re-verify all transactions, and there is no need for the normal nodes to re-verify the transactions.

Reply:
I’m up for discussing ideas about the strategy for rebroadcasting. My initial thought is that it should use a random amount of time since last rebroadcast as a threshold to determine whether to rebroadcast again for a transaction; also it should only consider rebroadcasting transactions that have a chance to make it in the next 2 blocks or so based on their sort order in the pool.

Reply:
@jsolman, I think it is a good idea to limit the re-broadcast for only covering the potential tx for the next 2-3 blocks:
https://github.com/neo-project/neo/blob/831003582b2ada4aff49244e5c88679b8982cc47/neo/Ledger/Blockchain.cs#L382-L398

Reply:
But, for that, we would need to know the order the possible order of a given  `tx` at `OnPersistCompleted`.
I think that it do not really need to be the exact order, but only to avoid sending some tx would be nice.

Reply:
Following the same reasoning, I think that honest Seed Nodes should not really send transactions that will surely not enter in the next block 3-5 blocks.

Reply:
What I mean is that only the consensus nodes need to re-verify `MaxTransactionsPerBlock` number of transactions immediately after the block is persisted. So this can be triggered by the SimplePolicy plugin, so you don't need to write these parameters to the core configuration file.

Reply:
@erikzhang 
Ok, I’ll look into adjusting to receive the params from the plugin. 

It would be nice if there were a way to inspect the maximum number of transactions each of the consensus nodes are configured to verify when they create a block. For now, perhaps the mechanism for determining whether to re-relay the reverified transactions will have to not consider how many transactions can be validated per block. I’ll come up with something that should ensure they don’t get orphaned and also doesn’t bog down the network if the mempool stays full for a long time.

Reply:
@erikzhang I've implemented reading the params from the plugin.